plugins/mongo_db/mongo_db_plugin.cpp

```diff
@@ -64,7 +65,13 @@ namespace mongo_db {
              "Write raw blocks into mongo or not")
             ("mongodb-write-operations",
              boost::program_options::value<std::vector<std::string>>()->multitoken()->zero_tokens()->composing(),
-             "List of operations to write into mongo");
+             "List of operations to write into mongo")
+            ("mongodb-store-dgp-history",
+             boost::program_options::value<int>()->default_value(10),
```

Comment:
Change type to `unsigned int` . 
For users it will be puzzle, how works negative numbers.